### PR TITLE
changes made on Top Navigation and car card

### PR DESCRIPTION
**Pull request by Pratishah**
deleted unnecessary style classes and comments ,
 removed duplicate test files ,
 changes on top nav styling

### CODE REVIEW

**Comment by Pratishah** [comment]
sorry i had to merge with dev instead of main branch

**Comment by pashedin** [comment]
this PR looks awfully similar to 
https://github.com/DteTRPod3/TRTraining_FinalAssignment/pull/26

**Comment by Pratishah** [comment]
Ticket Link:  https://abhirams.atlassian.net/browse/P3ECWFC-8
                    https://abhirams.atlassian.net/browse/P3ECWFC-10
                     https://abhirams.atlassian.net/browse/P3ECWFC-40

deleted unnecessary classnames , removed duplicate test files and comments, done styling on top Navbar and  Car-card
Screenshot (18)
Screenshot (20)

File: src/components/FeatureCars/FeaturedCars.scss [hunk 1]
```diff
@@ -7,39 +9,59 @@
...
+// .viewallbtn {
+//   a {
+//     float: right;
+//     margin-bottom: 5rem;
+//     display: block;
+//   }
+//   a:hover {
+//     color: $black;
+//   }
+// }
```
Comment (pashedin) [inline src/components/FeatureCars/FeaturedCars.scss:52]: You know what to do

File: src/components/Navbar/TopNavbar.tsx [hunk 2]
```diff
@@ -2,24 +2,44 @@ import React from "react";
...
+              >
+                ALL CARS
+              </Link>
+            </li>
+          </ul>
+        </div>
+        <div className="collapse navbar-collapse layout--header--container">
+          <ul className="navbar-nav mr-auto">
+            <li className="nav-item layout--header--list">
+              <Link to="" className="nav-link layout--header--link">
```
Comment (pashedin) [inline src/components/Navbar/TopNavbar.tsx:35]: please do not leave empty links, if there is nowhere to redirect to then add a `#`

**Review by pashedin — CHANGES_REQUESTED** [review]

**Pull request merged**